crates/bevy_render/src/render_phase/draw.rs

```diff
@@ -303,14 +218,16 @@ macro_rules! render_command_tuple_impl {
 all_tuples!(render_command_tuple_impl, 0, 15, C, V, E);
 
 /// Wraps a [`RenderCommand`] into a state so that it can be used as a [`Draw`] function.
-/// Therefore the [`RenderCommand::Param`] is queried from the ECS and passed to the command.
+/// Therefore the [`RenderCommand::Param`], [`RenderCommand::ViewWorldQuery`] and
```

Comment:
```suggestion
///
/// The [`RenderCommand::Param`], [`RenderCommand::ViewWorldQuery`] and
```

Again, not fond of the use of "therefore" without a prior clause to refer back to.